docs/rest_api.md

```diff
@@ -416,6 +416,29 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`PUT /node/<node>/boot_device`
+
+Sets the node's next boot device persistently
+
+The request body consists of JSON with a `bootdev` argument:
+
+Request body:
+    {
+
+    'bootdev': <boot device>
+
+    }
```

Comment:
Thanks! I have made these changes, now the tests are failing due to some other reason with our TravisCI.